impeller/tessellator/tessellator.cc

```diff
@@ -187,6 +187,9 @@ std::vector<Point> Tessellator::TessellateConvex(const Path& path,
                           [this](Path::Polyline::PointBufferPtr point_buffer) {
                             point_buffer_ = std::move(point_buffer);
                           });
+  if (polyline.points->size() == 0) {
```

Comment:
unrelated, right?

Reply:
@eyebrowsoffire @kevmoo  I didn't make any of these changes. Was a bit confusing to me that Github decided to include them as well. Maybe because I committed the work on top of the 3.22.2 branch instead of main? I can reapply the changes on top of main instead.

Reply:
Yes, I think you'll have to rebase on main to clean this up